Packs/CTIX/ReleaseNotes/2_1_0.md

```diff
@@ -0,0 +1,11 @@
+
+#### Integrations
+##### CTIX v3
+- Added new commands *domain*, *url* and *ip* and *file* which allow fetching of specfic SDO data from CTIX.
+
+
+<!-- 
+#### Integrations
+##### Cyware Threat Intelligence eXchange
+    - Did not make any changes, only combined descriptions into a single lines 
+-->
```

Comment:
This can be removed, customers will not see a difference :)

Reply:
Removed. The reason why I included this was because it wouldn't let me pass the validation checks without adding it. I guess it was detecting formatting changes from the other pack.